aws-sns-lambda/sns.tf

```diff
@@ -0,0 +1,25 @@
+locals {
+  default_sns_name = "${var.project}-${var.env}-${var.service}"
+}
+
+resource aws_sns_topic sns {
```

Comment:
**[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Missing version constraint for provider "aws" in "required_providers"